internal/sdkv2provider/resource_cloudflare_list.go

```diff
@@ -94,10 +94,6 @@ func resourceCloudflareListRead(ctx context.Context, d *schema.ResourceData, met
 	d.Set("description", list.Description)
 	d.Set("kind", list.Kind)
 
-	if !d.HasChange("items") && len(d.Get("item").(*schema.Set).List()) == 0 {
```

Comment:
this is to handle cases where the list items are handled externally - https://github.com/cloudflare/terraform-provider-cloudflare/pull/2295#discussion_r1139459258. have you tested that case here?

Reply:
Thank you for pointing me to that discussion, didn't see it, and will test it also.

Reply:
ok, i understand. Maybe we can to add some description about how that works, and also why I can't delete items if I delete them already from the cloudflare list resources?
Also, maybe we want to add some flag for addition check that if import run, that we want to add items to state?

Reply:
@jacobbednarz, I updated the MR, wdyt? Updating import will allow us to trigger read properly, and don't change any other logic outside of import.

Reply:
works for me!